add_pronoun_types.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+
+female = ['she', 'her', 'hers']
+subjective = ['she', 'he']
+objective = ['her', 'him']
```

Comment:
Will it cause problems that "her" is both possessive and objective?

Reply:
Very true, I guess only 'hers' can get that easily be identified as possessive. Maybe that's where the gender imbalance comes from.